.babelrc

```diff
@@ -8,7 +8,9 @@
           "last 2 Firefox versions",
           "last 2 Edge versions",
           "last 2 Safari versions"
-        ]
+        ],
+        // For uglifyjs
+        "uglify": true
```

Comment:
Is there any documentation for this key?

Reply:
https://github.com/babel/babel-preset-env#forcealltransforms

You can see the note here that `targets.uglify` is deprecated and will be removed in the next major version: https://github.com/babel/babel-preset-env/blob/84e12fcb3e929ca3f5edd65c3b03221a70f11881/README.md#forcealltransforms

If we are not using uglify to minify, we should remove it from our build step.